src/constants.js

```diff
@@ -0,0 +1,6 @@
+const constants = Object.freeze({
```

Comment:
I just don't feel that there is any real benefit to this, but it *will* make debugging more complicated.